### PR TITLE
Update events index

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Make it more like an agenda

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
screen shot 2018-10-01 at 12 59 36

Instead of https://www.defactolearning.de/veranstaltungskalender/

**Comment by RonaldDijkstra** [comment]
screen shot 2018-10-01 at 14 01 17

File: source/stylesheets/_evenementenagenda.scss [hunk 1]
```diff
@@ -64,34 +57,78 @@ body.evenementenagenda {
...
   .col {
-    @include pad();
-    vertical-align: middle;
+    display: block;
+    width: 100%;
+  }
 
-    p {
-      font-size: $base-font-size;
+  .left {
```
Comment (sn3p) [inline source/stylesheets/_evenementenagenda.scss]: Call this `.badge` maybe?

File: source/stylesheets/_evenementenagenda.scss [hunk 2]
```diff
@@ -64,34 +57,78 @@ body.evenementenagenda {
...
+      display: block;
+      height: 160px;
+      text-align: center;
+      width: 160px;
+    }
+
+    h2 {
+      color: $dark-gray;
+      font-size: $base-font-size * 4;
+      font-weight: 700;
```
Comment (sn3p) [inline source/stylesheets/_evenementenagenda.scss]: Maybe nice to make the space between the digits smaller:

```css
letter-spacing: -0.1em;
text-indent: -0.1em;
```

File: source/images/calendar-background.svg [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (sn3p) [inline source/images/calendar-background.svg]: Could [clean](https://jakearchibald.github.io/svgomg/) the SVG a little for smaller filesize.

File: source/stylesheets/_evenementenagenda.scss [hunk 4]
```diff
@@ -64,34 +57,78 @@ body.evenementenagenda {
...
+    }
+
+    h3 {
+      color: $white;
+      font-size: $base-font-size * 1;
+      font-weight: 700;
+      margin-bottom: 3px;
+      margin-top: 14px;
```
Comment (sn3p) [inline source/stylesheets/_evenementenagenda.scss]: Let CSS be in control of `uppercase`?

```css
text-transform: uppercase;
```

**Pull request merged**